test/helper.js

```diff
@@ -1,2 +1,21 @@
 import dotenv from 'dotenv';
 dotenv.config();
+
+import { git } from '../src/history/git.js';
+
+let lastCommitId;
+let stashResult;
+
+before(async () => {
+  const { latest: { hash } } = await git.log(['-1']);
+  lastCommitId = hash;
+  stashResult = await git.stash(['-u']);
+});
+
+after(async () => {
+  await git.reset(['--hard', lastCommitId]);
+  if (stashResult.includes('No local changes to save')) {
```

Comment:
Doesn't this rely on the host lang being set to `en`?

Reply:
Maybe but I'm not sure. I tried to change the lang on my machine and the sentence return by git was still in english.